### ISSUE TITLE
[question] Am I authorized to build a fully set-up Cross-Lib environment including ack?

### ISSUE THREAD

**Issue by Fabrizio-Caruso** [davidgiven/ack#236]
Am I authorized to build and distribute (freely) a fully set-up Cross-Lib environment including ack?
Cross-Lib is an open source and free (for no commercial use) cross-dev-kit library and tools to build games on several vintage systems by providing an abstraction layer:
https://github.com/Fabrizio-Caruso/CROSS-LIB

It can use CC65, Z88DK, CMOC, ACK, LCC1802, GCC and many others.
A full set-up takes too long for people who would just give it a try. So I would like to propose a docker image with all compilers and tools properly set-up.

Obviously I would provide links to the ACK project.

**Comment by davidgiven** [comment]
Yes, absolutely; provided the aggregate package is distributed under a license which meets all the terms of the ACK license, you can include it. You don't even have to ask.

I would add in terms of general licensing suggestions that it'd be good to make it clear that each of enclosed tools has their own individual license. Also, I see that one of your enclosed compilers is sdcc; this is GPL licensed and isn't compatible with your aggregate license, which looks like BSD 3-clause. I don't know about the others but that one I've met before.

**Comment by Fabrizio-Caruso** [comment]
@davidgiven Thanks!
The docker image won't contain *all* the supported tools and compilers but a significant sub-set.
sdcc is not needed even if Z88DK is included because sccz80 can do a decent job for both Intel 8080/5, Zilog 80 and GBZ80 CPUs.
So I won't include sdcc nor zsdcc.

I will only include tools and compilers that can run under Linux and that have compatible licenses.

By the way, please keep up your good with ACK! 
I have not really exploited it much because I am working on the other 200 targets and ACK comes with no much library support... I know it is out of scope for you... Anyway it is good to see a DOS target now.
Has anyone made an input/output library for DOS that can be used with ACK?
